src/transformers/utils/quantization_config.py

```diff
@@ -24,12 +24,17 @@
 
 from packaging import version
 
-from ..utils import is_auto_awq_available, is_torch_available, logging
+from ..utils import is_auto_awq_available, is_hqq_available, is_torch_available, logging
 
 
 if is_torch_available():
     import torch
 
+if is_hqq_available():
+    from hqq.core.quantize import BaseQuantizeConfig as HQQBaseQuantizeConfig
+else:
+    HQQBaseQuantizeConfig = None
```

Comment:
Can you import that at the init of `HqqConfig` so that for users that have `hqq` installed they won't have to import this module all the time even if they do not use it

Reply:
https://github.com/huggingface/transformers/pull/29637/commits/bc4bc73eb35cd87ef31ab292b7846c02c5ba3c22